offset_manager.go

```diff
@@ -366,7 +365,6 @@ func (pom *partitionOffsetManager) updateCommitted(offset int64, metadata string
 
 	if pom.offset == offset && pom.metadata == metadata {
 		pom.dirty = false
-		pom.offsetChangedSinceLastForced = true
```

Comment:
Removed this to simplify code. I think you are right @itszootime  code is simplier, and some extra commits once per hour would not harm